ngraph/core/include/ngraph/op/tensor_iterator.hpp

```diff
@@ -340,6 +337,8 @@ namespace ngraph
                 }
                 virtual void validate_and_infer_types() override;
                 void revalidate_and_infer_types_for_body_ops();
+                /// \return the body of the iteration
+                std::shared_ptr<Function> get_function() override;
```

Comment:
Looks like `get_function()` and `get_body()` do the same things. Can we remove `get_body()` method or deprecate it?

Reply:
get_body can be already in use. So to kepp backward compatibility I would keep as a separate method.